openforcefield/topology/topology.py

```diff
@@ -46,49 +47,65 @@
 #=============================================================================================
 
 # TODO: Can we have the `ALLOWED_*_MODELS` list automatically appear in the docstrings below?
-# TODO: Should these be the Python equivalent of enumerated types.
+# TODO: Should `ALLOWED_*_MODELS` be objects instead of strings?
+# TODO: Should these be imported from `openforcefield.cheminformatics.aromaticity_models` and `.bondorder_models`?
 
-DEFAULT_AROMATICITY_MODEL = 'MDL' # TODO: Is there a more specific spec for this?
+DEFAULT_AROMATICITY_MODEL = 'MDL' # TODO: Is there a more specific name and reference for the aromaticity model?
```

Comment:
OE MDL could work. Reference will be the SMIRNOFF paper, which will be where it is described relatively completely.

Reply:
@davidlmobley : Should this be called `OEMDL` instead, or is `MDL` sufficient?